tox.ini

```diff
@@ -4,7 +4,7 @@
 # directory.
 
 [tox]
-envlist = py27, py35, py36, py37, pypy
```

Comment:
Shouldn't py39 be added to match the Travis tests?

Reply:
@JamesJeffryes Sometimes people haven't wanted me to add the newest stable versions in Python. But I can add that.